nautobot/dcim/forms.py

```diff
@@ -3654,34 +3654,31 @@ class ConnectCableToRearPortForm(ConnectCableToDeviceForm):
 
 class ConnectCableToCircuitTerminationForm(ConnectCableExcludeIDMixin, NautobotModelForm):
     termination_b_provider = DynamicModelChoiceField(queryset=Provider.objects.all(), label="Provider", required=False)
-    termination_b_region = DynamicModelChoiceField(queryset=Region.objects.all(), label="Region", required=False)
-    termination_b_site = DynamicModelChoiceField(
-        queryset=Site.objects.all(),
-        label="Site",
+    termination_b_location = DynamicModelChoiceField(
```

Comment:
Hm. Something we missed supporting in 1.4/1.5? If we can't specify a Location when connecting a cable to a circuit-termination that's something we may need a backported fix for.